app/sitemap.njk

```diff
@@ -3,15 +3,10 @@ layout: page
 title: Sitemap
 permalink: "/sitemap.html"
 ---
-{% for item in collections.all | eleventyNavigation("app") %}
-  {% if loop.index == 1 %}
-    <h2 class="govuk-heading-l">Find</h2>
-  {% elseif loop.index == 3 %}
-    <h2 class="govuk-heading-l">Apply</h2>
-  {% endif %}
-  <h3 class="govuk-heading-m govuk-!-margin-bottom-1">
+{% for item in collections.all | eleventyNavigation("home") %}
```

Comment:
What does this bit: `eleventyNavigation("home")` do here?

Reply:
It does a couple of things. First, it only shows pages below `home` (it doesn’t include a link to the homepage in the list of pages). It also allows us to use a different presentation for the top level items to any sub pages they may have, which appear nested

<img width="640" alt="Screenshot 2020-06-17 at 10 57 09" src="https://user-images.githubusercontent.com/813383/84884416-97ae1d80-b089-11ea-8848-e87abb2b1c0a.png">

<img width="480" alt="Screenshot 2020-06-17 at 10 57 37" src="https://user-images.githubusercontent.com/813383/84884420-9a107780-b089-11ea-99cd-1db25a5278f5.png">

.